libs/ledger-live-common/src/families/tron/bridge/js.ts

```diff
@@ -567,7 +619,7 @@ const getTransactionStatus = async (a: TronAccount, t: Transaction): Promise<Tra
     errors.recipient = new RecipientRequired();
   }
 
-  if (["send", "freeze", "unfreeze"].includes(mode)) {
```

Comment:
Is the `freeze` case not handled anymore?

Reply:
Yeah, it seemed unnecessary since we don't use `recipient` in the new `FreezeTransactionData` and `UnFreezeTransactionData`.